config/packages/db_tools.yaml

```
@@ -1,77 +1,62 @@
 db_tools:
     # Where to put generated backups.
-    #storage:
-        # Root directory of the backup storage manager. Default filename
-        # strategy will always use this folder as root path.
-        #root_dir: '%kernel.project_dir%/var/db_tools'
+    # Root directory of the backup storage manager. Default filename
+    # strategy will always use this folder as root path.
+    storage_directory: '%kernel.project_dir%/var/db_tools'
```

If this is default value then:

```suggestion
    #storage_directory: '%kernel.project_dir%/var/db_tools'
```
if not, ignore this comment